select2-parent/select2/src/main/java/org/wicketstuff/select2/AbstractSelect2Choice.java

```diff
@@ -465,4 +465,12 @@ public void onComponentTagBody(final MarkupStream markupStream, final ComponentT
         }
 		replaceComponentTagBody(markupStream, openTag, buffer);
 	}
+	
+	/**
+	 * Empty input is acceptable
+	 */
+	@Override
+	public boolean isInputNullable() {
+		return true;
+	}
```

Comment:
@solomax Imho, it's not related to mentioned by you link. isInputNullable was overriden in AbstractTextComponent which was used as base class for AbstractSelect2Choice where, I think, it should be overriden one more time. And btw, In AbstractSingleSelectChoice method isInputNullable returns true, as it's done by this PR
@martin-g please advise what is correct.